cmd/src/args/user.rs

```diff
@@ -132,3 +173,107 @@ pub(crate) struct UserListArgs {
     #[clap(short, long, value_enum, default_value_t = ListMode::Table)]
     pub(crate) list_mode: ListMode,
 }
+
+#[derive(Debug, Clone, Args)]
+pub(crate) struct UserNameArgs {
+    /// User ID to update
+    ///
+    /// The user ID can be specified as either a username or an ID
+    pub(crate) user_id: Identifier,
+    /// New username
```

Comment:
Maybe we could specify here the basic validation rules, being 3-50 characters (and same for the password which is 3-100 chars)? No other requirements regarding the casing or special symbols for now.